CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+5.0.0
```

Comment:
does not need to be major i think

Reply:
2.7 is out of live, you're using it at your own risk

Reply:
Agreed. I did the same thing in https://github.com/gocardless/anony/pull/101 and after Kevin pointed out that it possibly doesn't need to be a major revision I did some reading and came to the conclusion that a major version bump should only be for breaking API changes because bundler can figure out that `4.5.0` isn't compatible with `4.4.0` if you're using ruby 2.7 still and so won't upgrade

This article helped me come to the decision https://medium.com/@sampart/semantic-versioning-when-you-change-the-required-programming-language-version-16a3a3555c95

Reply:
Fair enough, have reverted to just a minor bump.